benches/insert.rs

```diff
@@ -0,0 +1,47 @@
+#![feature(test)]
+
+extern crate test;
+extern crate rle_vec;
+
+use std::iter::FromIterator;
+use std::iter::repeat;
+use test::Bencher;
+use rle_vec::RleVec;
+
+#[bench]
+fn rle_insert_middle_non_breaking_10_000_runs_of_10_values(b: &mut Bencher) {
+    b.iter(|| {
+        let zeros = repeat(0).take(10);
+        let ones = repeat(1).take(10);
+        let iter = repeat(zeros.chain(ones)).flat_map(|x| x).take(10_000);
+
+        let mut rle = RleVec::from_iter(iter);
+        let middle_value = rle[5_000];
+        rle.insert(5_000, middle_value); // ???
+    })
+}
+
+#[bench]
+fn rle_insert_middle_breaking_10_000_runs_of_10_values(b: &mut Bencher) {
+    b.iter(|| {
+        let zeros = repeat(0).take(10);
+        let ones = repeat(1).take(10);
+        let iter = repeat(zeros.chain(ones)).flat_map(|x| x).take(10_000);
+
+        let mut rle = RleVec::from_iter(iter);
+        rle.insert(5_000, 424242); // ???
```

Comment:
This only breaks the run in the first iteration unless you change the inserted value. This became clear in the bench results.

Also all these functions use 1000 runs of 10 values. Not 10000 as the function name implies.